test/action.js

```diff
@@ -173,6 +173,45 @@ describe('Action', function() {
 			siren = buildAction();
 			expect(siren.getField('foo')).to.have.property('title', 'bar');
 		});
+
+		it('should be able to use field helper methods', function() {
+			resource.fields = [{
+				class: ['abc'],
+				name: 'foo',
+				title: 'bar'
+			}];
+			siren = buildAction();
+			expect(siren.fields[0].hasClass('abc')).to.be.true;
+		});
+	});
+
+	describe('toJSON', function() {
+		function toJSON() {
+			return JSON.stringify(buildAction());
+		}
+
+		it('should stringify name and href', function() {
+			expect(toJSON()).to.equal(
+				'{"name":"foo","href":"bar","method":"GET","type":"application/x-www-form-urlencoded"}'
```

Comment:
If you want to make these more specific, mocha does have `match`, so you can specifically test for the name/href here with something like

```
expect(toJSON()).to.match(/\{.*"name":"foo".*\}/);
expect(toJSON()).to.match(/\{.*"href":"bar".*\}/);
```